### PR TITLE
Flow

### PR DESCRIPTION
**Pull request by alex-wilmer**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,25 +8,26 @@
...
+    "recompose": "^0.23.1",
+    "styled-components": "2.0.0-15"
   },
   "homepage": "https://torontojs.github.io",
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "deploy": "npm run build&&gh-pages -d build",
-    "test": "eslint src",
+    "test": "eslint src; flow",
```
Comment (tazsingh) [inline package.json]: Not `eslint src && flow`? :P This is what I do in my projects to be a douche to my colleagues

Comment (alex-wilmer) [inline package.json]: In both cases Travis will fail if something is wrong, but with the `;` case you'll at least see which of the two, or both have failed. Is there a negative to `;` I'm not thinking about?

Comment (tazsingh) [inline package.json]: Really? Travis will fail if either of the commands return a non 0 status code? TIL.

All is find here then.

Comment (alex-wilmer) [inline package.json]: I'm going to double-check before merging. If i'm wrong I'll change it to `&&`

**Comment by alex-wilmer** [comment]
@tazsingh oops guess not!

**Pull request merged**